oonidata/observations.py

```diff
@@ -305,14 +309,14 @@ def from_measurement(
             dnso.answer = answer.hostname
 
         if answer.ipv4 or answer.ipv6:
-            answer_meta = netinfodb.lookup_ip(dnso.timestamp, dnso.answer)
+            answer_meta = netinfodb.lookup_ip(dnso.timestamp, dnso.answer)  # type: ignore
```

Comment:
Same as above: I'd like the codebase to document why we're ignoring some types. This would allows us to make these `type: ignore` comment hopefully actionable at a future time.